src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/patch/ObjectionPatch.java

```diff
@@ -2,9 +2,12 @@
 
 import uk.gov.companieshouse.api.strikeoffobjections.model.entity.ObjectionStatus;
 
+import java.time.LocalDateTime;
+
 public class ObjectionPatch {
     private String reason;
     private ObjectionStatus status;
+    private LocalDateTime createdOn;
```

Comment:
could we just have the patch method set the current timestamp, just thinking that we could set a status with last years date if we wanted